_replication/replication-methods/log-based-incremental-replication.md

```diff
@@ -85,10 +85,13 @@ sections:
           2. Stitch will use a `SELECT`-based approach to replicate the table in full. If this were a SQL query, it would look like this:
 
              ```sql
-             SELECT column_you_selected_1,
+             SELECT id,
+                    column_you_selected_1,
                     column_you_selected_2,
                     [...]
                FROM schema.table
+              WHERE id < [max id value]
+              ORDER BY id
```

Comment:
@cmerrick I don't think it's entirely obvious what happens if a historical sync spans multiple jobs. Is this accurate?:

1. Historical sync starts; max PK for the table is stored
2. Historical sync doesn't complete during job; bookmark for table is stored
3. Next job starts, historical sync resumes using bookmark
4. Jobs continue replicating historical data using bookmark until complete
5. When historical is complete, switch to logs

I also thought this would only be used if the PK met certain requirements - if that's the case, we should include it here.

Reply:
I added a note about omitting the WHERE clause when the PK doesn't meet requirements. Your description of the multi-job scenario is accurate, but IMO is too complicated and too much of an edge case to include in these docs.